en/config/actuators.md

```diff
@@ -0,0 +1,140 @@
+# Actuator Configuration and Testing
+
+:::note
```

Comment:
Moved up top so people immediately know why it isn't displayed. We just delete this note when the thing goes default enabled.